pkg/projectfile/projectfile.go

```diff
@@ -822,17 +822,20 @@ func GetProjectFilePath() (string, *failures.Failure) {
 		getProjectFilePathFromWd,
 		getProjectFilePathFromDefault,
 	}
+	var path string
+	var fail *failures.Failure
 	for _, getProjectFilePath := range lookup {
-		path, fail := getProjectFilePath()
+		path, fail = getProjectFilePath()
 		if fail != nil {
-			return "", fail
+			logging.Debug("Failed to find project file path: %v", fail)
```

Comment:
This would change our current control flow:  If an ACTIVESTATE_PROJECT environment variable is set, but the file is not found, it would not be an error anymore.  @Naatan convinced me in a different PR, that we probably shouldn't do that.

So, is the error actually because we didn't write the default config path early enough to the config file?  In that case, adding the `cfg.WriteConfig()` to the global default setup routine should already take care of the problem, no?

Reply:
Yes, confirmed that updating `cfg.WriteConfig()` with the original loop will work.